kubeadm/v1.15.0/Kubeclustervxlan.json

```diff
@@ -0,0 +1,44 @@
+{
+    "Cri" : {
+        "Name" : "dockerd",
+        "Images" : {
+            "Pause" : "mcr.microsoft.com/k8s/core/pause:1.0.0",
+            "Nanoserver" : "mcr.microsoft.com/windows/nanoserver:1809",
+            "ServerCore" : "mcr.microsoft.com/windows/servercore:ltsc2019"
+        }
+    },
+    "Cni" : {
+        "Name" : "flannel",
+        "Source" : [{ 
+            "Name" : "flanneld",
+            "Url" : "https://github.com/coreos/flannel/releases/download/v0.11.0/flanneld.exe"
+            }
+        ],
+        "Plugin" : {
+            "Name": "vxlan"
+        },
+        "InterfaceName" : "Ethernet"
+    },
+    "Kubernetes" : {
+        "Source" : {
+            "Release" : "1.14.0",
+            "Url" : "https://dl.k8s.io/v1.14.0/kubernetes-node-windows-amd64.tar.gz"
+        },
+        "Master" : {
```

Comment:
This needs to be renamed `ControlPlane` to pass `if (!$Global:ClusterConfiguration.Kubernetes.ControlPlane)` in the script